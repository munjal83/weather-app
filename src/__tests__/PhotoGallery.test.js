import React from 'react';
import ReactDOM from 'react-dom';
import { create } from 'react-test-renderer';
import PhotoGallery from '../components/PhotoGallery';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<PhotoGallery />, div);
  ReactDOM.unmountComponentAtNode(div);
});

describe('PhotoGallery Component Snapshot',()=>{
  test('testing photogallery component', () => {
  let tree = create(<PhotoGallery />)
  expect(tree.toJSON()).toMatchSnapshot();
})
});