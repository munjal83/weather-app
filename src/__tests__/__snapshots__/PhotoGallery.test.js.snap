// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoGallery Component Snapshot testing photogallery component 1`] = `
<div>
  <div>
    Â 
  </div>
</div>
`;
